src/website/apps/blog/tests.py

```
@@ -231,4 +244,675 @@ def test_calendar_endpoint_handles_legacy_quill_data(self):
+            event=event, partner=partner, role='partner', order=1
+        )
+        self.assertEqual(link.event, event)
+        self.assertEqual(link.partner, partner)
+        self.assertEqual(link.role, 'partner')
+        self.assertEqual(event.event_partner_links.count(), 1)
+
+    def test_duplicate_partner_link_is_rejected(self):
+        event = self._make_event('Test Event')
+        partner = self._make_partner('Google')
+        EventPartner.objects.create(event=event, partner=partner)
+        with self.assertRaises(Exception):
```

Ruff (0.15.15)

[warning] Do not assert blind exception: `Exception`

(B017)